dimagi/commcare-hq
# USH-4622: Fix repeat group accessiblity and clean up after FP deploy

## Product Description

Make "add group" button part of the group template. This fixes an issue with tabbing through questions to the "add group" button.

## Technical Summary

https://dimagi.atlassian.net/browse/USH-4622

## Feature Flag

case_claim_autolaunch

## Safety Assurance

### Safety story
 
Tested locally and on staging + QA

### Automated test coverage

Updated tests

### QA Plan

Should cover previous repeat group test and broadcasting questions. I already added a form [here](https://staging.commcarehq.org/a/qateam/apps/view/388693e7e83fccfdd08356b35da2d58b/form/0e39015b90f24fae886edf95da5e62b5/source/#form/address_outside_groups) that should cover all scenarios.

### Rollback instructions




### Labels & Review
